src/components/Navbar/MenuItems.tsx

```diff
@@ -98,13 +98,15 @@ export function TopLeftNavItems({
         text={t`FAQ`}
         route={undefined}
         onClick={() => {
-          if (onClickMenuItems) onClickMenuItems()
-          window.location.hash = '/'
-          setTimeout(() => {
-            document
-              .getElementById('faq')
-              ?.scrollIntoView({ behavior: 'smooth' })
-          }, 0)
+          if (typeof window !== 'undefined') {
```

Comment:
**nitpick (non-blocking)** super minor, but I find it more readable to avoid nesting code in this block and return early using the inverse logic.

```js
if (typeof window === 'undefined') {
    return
}
// ...code
```

Like this: https://github.com/jbx-protocol/juice-interface/pull/1366/files#diff-8c58fee43e99a62af46f20eba09c7918f17b7d4fc16e6822e5fa566fb7c49bb8R59-R61